commands/team.go

```diff
@@ -305,20 +307,24 @@ func deleteTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	var result *multierror.Error
+
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
 			printer.PrintError("Unable to find team '" + args[i] + "'")
+			result = multierror.Append(result, fmt.Errorf("unable to find team %s", args[i]))
 			continue
 		}
 		if _, err := deleteTeam(c, team); err != nil {
 			printer.PrintError("Unable to delete team '" + team.Name + "' error: " + err.Error())
+			result = multierror.Append(result, fmt.Errorf("unable to delete team %s error: %s", team.Name, args[i]))
```

Comment:
Please use `%w` for errors when using `fmt.Errorf`, that way the error is preserved (for possible future use).

Reply:
Thank you for the suggestion. I did, but the tests failed, and I got this message

```
commands/team.go:316:39: fmt.Errorf format %w has arg args[i] of wrong type string
commands/team.go:321:39: fmt.Errorf format %w has arg team.Name of wrong type string
```

Reply:
Oh, I assumed you were logging `err` in the message, rather than `args[1]`.
```suggestion
			result = multierror.Append(result, fmt.Errorf("unable to delete team %s error: %w", team.Name, err))
```